lib/discourse_activity_pub/follow_handler.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+module DiscourseActivityPub
+    class FollowHandler
+        attr_reader :actor,
+                    :target_actor
+
+        def initialize(actor_id, target_actor_id)
+            @actor = DiscourseActivityPubActor.find_by_id(actor_id)
+            @target_actor = DiscourseActivityPubActor.find_by_id(target_actor_id)
+        end
+
+        def follow
+            return false unless actor && target_actor&.remote?
```

Comment:
Ah this looks like the answer to my earlier comment, we can only follow if target actor is remote, cool!

Reply:
👍